rdr_service/dao/site_hierarchy_dao.py

```diff
@@ -0,0 +1,84 @@
+from typing import Dict
+
+from flask import jsonify, Response
+from sqlalchemy import or_
+from sqlalchemy.orm import Session, query
+from werkzeug.exceptions import BadRequest
+
+from rdr_service.dao.base_dao import BaseDao
+from rdr_service.model.site import Site
+from rdr_service.model.organization import Organization
+from rdr_service.model.hpo import HPO
+from rdr_service.model.site_enums import ObsoleteStatus
+
+
+class SiteHierarchyDao(BaseDao):
+    _QUERY_PARAM_MAPPING = {
+        "awardee_id": {"col": HPO.name},
+        "organization_id": {"col": Organization.externalId},
+        "google_group": {"col": Site.googleGroup},
+    }
```

Comment:
:nit Can we also flatten the values in this map? If you had more than one key in the dictionaries I would understand but I think it would be better to default it to the model attribute instead to make the dict accessing cleaner.
`_QUERY_PARAM_MAPPING.get(key)`

Reply:
sure. I renamed the variable to `_QUERY_PARAM_TO_COLUMN_MAPPING`, so its clear the values in the dict are column names in model classes.